client/js/screens/MyCampaigns/MyCampaigns.js

```diff
@@ -12,62 +12,44 @@ import styles from './styles';
 import spaceTexture from '../../assets/Textures/DarkSpace.png';
 import SmallUnsubscribeButton from '../../components/SmallUnsubscribeButton';
 import ActiveButton from '../../assets/buttons/ButtonSPace.png';
-const User = [];
-const DATA = [
-  {
-    id: '1',
-    title: 'International Rebellion',
-    category: 'Enviornment',
-  },
-  {
-    id: '2',
-    title: 'Peace and Humanity',
-    category: 'Political',
-  },
-  {
-    id: '3',
-    title: 'Unity and Diversity',
-    category: 'Social',
-  },
-  {
-    id: '4',
-    title: 'Power Politics in Todays World',
-    category: 'Social',
-  },
-];
+import {withNavigation} from 'react-navigation';
 
-const Item = ({title, category}) => {
+const MyCampaigns = ({myCampaigns, user, navigation}) => {
   return (
-    <View style={styles.item}>
-      <View>
-        <Text style={styles.title}>{title}</Text>
-        <Text style={styles.category}>{category}</Text>
-      </View>
-      <SmallUnsubscribeButton />
-    </View>
-  );
-};
-
-const MyCampaigns = props => {
-  return (
-    <>
+    <SafeAreaView style={styles.container}>
       <Text style={styles.pageTitle}>My Campaigns </Text>
-      {DATA && DATA.length > 0 ? (
+      {myCampaigns && myCampaigns.length > 0 ? (
         <ImageBackground
-          style={styles.panel}
+          style={styles.background}
           imageStyle={styles.panel}
           source={spaceTexture}>
-          <SafeAreaView style={styles.container}>
-            <FlatList
-              data={DATA}
-              renderItem={({item}) => (
-                <TouchableOpacity>
-                  <Item title={item.title} category={item.category} />
+          <FlatList
+            data={myCampaigns}
+            renderItem={({item}) => (
+              <View style={styles.item}>
+                <TouchableOpacity
+                  style={{
+                    flex: 1,
+                    flexDirection: 'row',
+                    justifyContent: 'space-between',
+                    height: '100%',
+                  }}
+                  onPress={() => {
+                    navigation.navigate('Campaign', {
+                      campaign: item,
+                    });
+                  }}>
+                  {/* View for text flexDirection = column (auto)*/}
```

Comment:
Please move all the inline styles to styles.js file and remove the commented code if not needed.

Reply:
Done.